packages/bitcore-wallet-service/src/lib/model/wallet.ts

```diff
@@ -56,7 +51,7 @@ export class Wallet {
   n: number;
   singleAddress: boolean;
   status: string;
-  publicKeyRing: string[];
+  publicKeyRing: any[];
```

Comment:
why any?

Reply:
Here, if we do this:
```this.publicKeyRing = _.map(this.copayers, (copayer) => {
      return _.pick(copayer, ['xPubKey', 'requestPubKey']);
    });```

We get something like:  `Array<{xPubKey: string, requestPubKey: string}>`
so `string[]` is not quite right .. That’s why I had put `any[]`

We could use `Array<{xPubKey: string, requestPubKey: string}>` to be more specific 

^ Done